crates/matrix-sdk/Cargo.toml

```
@@ -31,6 +32,7 @@ js = ["matrix-sdk-common/js", "matrix-sdk-base/js"]
 
 sled = ["dep:matrix-sdk-sled", "matrix-sdk-sled?/state-store"]
 indexeddb = ["dep:matrix-sdk-indexeddb"]
+redis = ["dep:matrix-sdk-redis", "matrix-sdk-redis?/crypto-store"]
```

```suggestion
redis = ["dep:matrix-sdk-redis"]
```

The `crypto-store` feature should be enabled via the `e2e-encryption` feature above.